src/common/SPELLS/monk.ts

```diff
@@ -471,6 +471,12 @@ const spells: SpellList = {
     name: "Hot Trub",
     icon: "spell_brew_dark",
   },
+  // Conduits
+  EVASIVE_STRIDE_HEAL: {
```

Comment:
We've been putting these in the conduit file common/spells/shadowlands/conduits/monk.ts